Formula/sqlancer.rb

```diff
@@ -0,0 +1,41 @@
+class Sqlancer < Formula
+  desc "Detecting Logic Bugs in DBMS"
+  homepage "https://github.com/sqlancer/sqlancer"
+  url "https://github.com/sqlancer/sqlancer/archive/1.1.0.tar.gz"
+  sha256 "ce36d338e7af3649256ff1af89d26ca59fee8e8965529c293ef5592e103953fc"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Is this actually needed @samford

Reply:
I just saw it in a couple of other formulae.
Actually, in about 137 other formulae:
```
$ git grep -l ':github_latest' | xargs grep '/archive/' | wc -l
     137
```

Reply:
I think the idea is to assume it's not needed, then just add it later on if it turns out it is: https://github.com/Homebrew/brew/blob/master/docs/Brew-Livecheck.md#url-guidelines

Reply:
Ok, I've deleted `livecheck` block.

Reply:
> I think the idea is to assume it's not needed, then just add it later on if it turns out it is

Yes, in general, we only add a `livecheck` block if the default check isn't correct. Our position on this may change sometime in the future (for technical reasons) but this is how we currently approach things, as it reduces maintenance.

> Is this actually needed

Without the `livecheck` block, this will check the Git tags from the GitHub repository by default, which gives `1.1.0` as newest. This is the same version as "latest", so we don't need a `livecheck` block for this formula. Currently we only add one if the default check isn't sufficient/correct.

We also [restrict use of the `GithubLatest` strategy](https://docs.brew.sh/Brew-Livecheck#general-guidelines) to when it's both correct and necessary to identify the latest version. That is to say, if the Git strategy was returning a newer version than "latest" on GitHub (e.g., a tag that's not yet a release, a pre-release version, etc.) and it couldn't be corrected with a regex, we would have to use `GithubLatest`. We avoid using the `GithubLatest` strategy unless it's necessary, as github.com is rate limited and can potentially lead to an unexpected CI failure if we have too many checks hitting the website.